operator/controllers/lifecycle/keptntask/job_utils.go

```diff
@@ -79,6 +60,11 @@ func (r *KeptnTaskReconciler) createFunctionJob(ctx context.Context, req ctrl.Re
 	return job.Name, nil
 }
 
+func emptySpec(definition *klcv1alpha3.KeptnTaskDefinition) bool {
```

Comment:
shouldn't this function be in the container_builder.go? I think here we will add additional logic about builders in the future, which can be then stored there on "common" places for all builders

Reply:
yes, well spotted!